EdgeCraftRAG/Dockerfile.ui

```diff
@@ -0,0 +1,23 @@
+FROM python:3.11-slim
```

Comment:
Align with other code structure, the ui dockerfile should be under, `ui/docker/xxx`

Reply:
resolved